src/Components/Board.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
```

Comment:
I would change the name of this folder to `components`

I would only ever Capitalize a file/folder name if it were for a React component, or a class or something